merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/GetSimulationResultsAction.java

```diff
@@ -78,7 +77,7 @@ public Map<String, List<Pair<Duration, SerializedValue>>> getResourceSamples(fin
     final var revisionData = this.planService.getPlanRevisionData(planId);
     return this.simulationService
         .get(planId, revisionData)
-        .map(r -> r.resourceSamples)
+        .map(SimulationResults::resourceSamples)
```

Comment:
Could you clarify where `resourceSamples` gets the `SimulationResults` parameter it needs here?

Reply:
Sure thing - `map` provides the simulation results as an argument - the code below is equivalent: 

```suggestion
        .map(results -> SimulationResults.resourceSamples(results))
```

Reply:
Thanks!